src/pages/dashboard/index.tsx

```diff
@@ -0,0 +1,244 @@
+        return "bg-sky-100 text-sky-600";
+      case ActivityStatus.Success:
+        return "bg-emerald-50 text-emerald-500";
+    }
+  }, [status]);
+  return (
+    <Badge className={clsx(className, "flex items-center")}>
+      {t(capitalize(status))}
+    </Badge>
+  );
+};
+
+const dummy = [
+  {
+    description: "All datasources are up to date!",
+    occurredAt: "2022-11-09T00:00:39.651Z",
+    url: "/catalog/",
+    status: "ERROR",
+  },
+  {
+    description: "Quentin Did that",
+    occurredAt: "2022-11-09T00:00:39.651Z",
+    url: "/catalog/",
+    status: "ERROR",
+  },
+];
+const DashboardPage = () => {
+  const { t } = useTranslation();
+  const { data } = useDashboardPageQuery();
+
+  return (
+    <Page title={t("Dashboard")}>
+      <PageContent>
+        <Breadcrumbs className="my-8 px-2">
+          <Breadcrumbs.Part href="/pipelines">
+            {t("Dashboard")}
+          </Breadcrumbs.Part>
+        </Breadcrumbs>
+        <div className="my-12">
+          <div className="flex items-center justify-between">
+            <h3 className="ml-2 mt-2 text-lg leading-6 text-gray-900">
+              {t("Overview")}
+            </h3>
+          </div>
+          <div className="mt-4">
+            <div className="grid grid-cols-1 gap-5 sm:grid-cols-2 lg:grid-cols-3">
+              <Block>
+                <div className="p-5">
+                  <div className="flex items-center">
+                    <div className="flex-shrink-0">
+                      <BookmarkSquareIcon className="text-cool-gray-400 h-6 w-6" />
+                    </div>
+                    <div className="ml-5 w-0 flex-1">
+                      <DescriptionList
+                        displayMode={DescriptionListDisplayMode.LABEL_ABOVE}
+                      >
+                        <DescriptionList.Item
+                          label={
+                            <div className="text-cool-gray-500 truncate text-sm font-medium leading-5">
+                              {t("Datasources")}
+                            </div>
+                          }
+                        >
+                          <div className="text-cool-gray-900 text-lg font-medium leading-7">
+                            {data?.catalog.totalItems}
+                          </div>
+                        </DescriptionList.Item>
+                      </DescriptionList>
+                    </div>
+                  </div>
+                </div>
+                <div className="bg-cool-gray-50 px-5 py-3">
+                  <div className="text-sm leading-5">
+                    <Link
+                      href="/catalog"
+                      className="font-medium text-blue-600 transition duration-150 ease-in-out hover:text-blue-900"
+                    >
+                      {t("View all")}
+                    </Link>
+                  </div>
+                </div>
+              </Block>
+              <Block>
+                <div className="p-5">
```

Comment:
Why not use className on Block directly ?

Reply:
Okay